pkg/ipmi/dev.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ipmi
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+type Dev struct {
```

Comment:
we should decide whether to export or unexport Dev and then adjust the API accordingly. As is, Dev is not a useful exported type, because it's API is mixed: E.g. Dev is exported, SendRequest is exported, but it takes an unexported argument of `request`, so external callers cannot actually use this API.

Either make a case for why this API is useful to external callers, or let's just unexport it. (My vote for now is to unexport it. We can decide it's useful for others later.)

Reply:
You marked this conversation as resolved, but I don't see any changes? Forget to upload?

Reply:
Jip.